src/screen.js

```diff
@@ -1,23 +1,41 @@
+import {compressToEncodedURIComponent} from 'lz-string'
 import * as queries from './queries'
 import {getQueriesForElement} from './get-queries-for-element'
 import {logDOM} from './pretty-dom'
+import {getDocument} from './helpers'
+
+function unindent(string) {
+  // remove white spaces first, to save a few bytes.
+  // testing-playground will reformat on load any ways.
+  return (string || '').replace(/[ \t]*[\n][ \t]*/g, '\n')
+}
+
+function encode(value) {
+  return compressToEncodedURIComponent(unindent(value))
+}
+
+function getPlaygroundUrl(element) {
+  return `https://testing-playground.com/#markup=${encode(element.innerHTML)}`
+}
 
 const debug = (element, maxLength, options) =>
   Array.isArray(element)
     ? element.forEach(el => logDOM(el, maxLength, options))
     : logDOM(element, maxLength, options)
 
+const logTestingPlaygroundURL = (element = getDocument().body) => {
+  console.log(`Open this URL in your browser\n\n${getPlaygroundUrl(element)}`)
+}
```

Comment:
As for the previous comment. What do you think of this?

```suggestion
const logTestingPlaygroundURL = (element = getDocument().body) => {
  if ('innerHTML' in element) {
    if (element.innerHTML === '')
      console.log(
        `The provided element(${element.nodeName}) doesn't have any children.`,
      )
    console.log(`Open this URL in your browser\n\n${getPlaygroundUrl(element)}`)
  } else {
    console.log(`The element you're providing isn't a valid DOM element.`)
  }
}